admin_manual/enterprise_user_management/file_firewall.rst

```diff
@@ -0,0 +1,174 @@
+Group** button. After setting up your rules you must click the **Save Rules** 
+button.
+
+.. figure:: images/firewall-1.png
+   :alt: Empty File Firewall configuration panel.
+   
+   *Figure 1: Empty File Firewall configuration panel*
+
+Figure 2 shows two rules. The first rule, **No Support outside 
+office hours**, prevents members of the support group from logging into the 
+ownCloud Web interface from 5pm-9am.
+
+The second rule prevents members of the qa-team group from accessing the Web UI 
+from IP addresses that are outside of the local network.
+
+.. figure:: images/firewall-2.png
+   :alt: Two example rules that restrict logins per user group.
+   
+   *Figure 2: Two example rules that restrict logins per user group*   
+
+All other users are not affected, and can log in anytime from anywhere.
+
+Available Conditions
+--------------------
+
+User Group
+ The user (is|is not) a member of the selected group.
+
+User Agent
+ The User-Agent of the request (matches|does not match) the given string.
+
+User Device
+  A shortcut for matching all known (`android`|`ios`|`desktop`) sync clients by 
+  their User Agent string.
+
+Request Time
+ The time of the request (has to|must not) in a single range from beginning 
+ time to end time.
+
+Request URL
+ The **full page URL** (has to|must not) (match|contain|begin with|end) with a 
+ given string.
+
+Request Type
+ The request (is|is not) a (WebDAV|public share link|other) request.
+
+Request IP Range
+ The request's ``REMOTE_ADDR`` header (is|is not) matching the given IP range.
+
+Subnet
+ The request's ``SERVER_ADDR`` header (is|is not) matching the given IP range.
+
+File Size Upload
+ When a file is uploaded the size has to be (less|less or equal|greater|greater 
+ or equal) to the given size.
+
+File Mimetype Upload
+ When a file is uploaded the mimetype (is|is not|begins with|does not begin 
+ with|ends with|does not end with) the given string.
+
+Regular Expression
+ The File Firewall supports regular expressions, allowing you to create custom 
+ rules using the following conditions:
+
+* User IP
+* Subnet
+* User agent
+* User group
+* Request URL
+
+You can combine multiple rules into one rule. E.g., if a rule applies to both 
+the support and the qa-team you could write your rule like this::
+
+ Regular Expression > `^(support|qa-team)$` > `is` > `User group`
+
+No Manual Editing
+-----------------
+
+We do not recommend modifying the configuration values directly in your
+``config.php``. These use JSON encoding, so the values are difficult to read 
+and a single typo will break all of your rules.
+
+Correctly Blocking Access to a Folder
+-------------------------------------
+
+See examples of the correct regexp syntax to use for controlling access to 
+specific folders at
+`Example rule to restrict access to a folder based on 
+CIDR <https://github.com/owncloud/firewall/issues/53#issuecomment-141995297>`_.
```

Comment:
eh phooey, no shortcut for the overworked tech writer :) Yes, they must be copied.